projects/packages/calypsoify/src/class-jetpack-calypsoify.php

```diff
@@ -68,11 +69,17 @@ public function setup() {
 	 */
 	public function enqueue_for_gutenberg() {
 		$site_suffix = ( new Status() )->get_site_suffix();
-		wp_enqueue_style( 'calypsoify_wpadminmods_css', plugin_dir_url( __FILE__ ) . 'build/style-gutenberg.css', false, self::PACKAGE_VERSION );
-		wp_style_add_data( 'calypsoify_wpadminmods_css', 'rtl', 'replace' );
-		wp_style_add_data( 'calypsoify_wpadminmods_css', 'suffix', '.min' );
 
-		wp_enqueue_script( 'calypsoify_wpadminmods_js', plugin_dir_url( __FILE__ ) . 'build/mods-gutenberg.js', array( 'jquery' ), self::PACKAGE_VERSION, false );
+		Assets::register_script(
+			'calypsoify_wpadminmods_js',
+			'../dist/index.js',
+			__FILE__,
+			array(
+				'jquery',
```

Comment:
I think you meant like this here
```suggestion
				'dependencies' => array( 'jquery' ),
```

OTOH, if you were to add an `import jQuery from 'jquery';` to the JS file, then it would show up in the `index.asset.php` file and you could omit specifying it here.

But if you're going to update the script, even better would probably be to replace the `$( 'some selector' )` with `document.querySelector()` or `document.querySelectorAll()`, and then the `.attr()` calls with direct accesses to `.href` and `.target`, so we can avoid requiring jquery at all. 🙂 And probably also replace the uses of `wp.data` with `import { dispatch, select } from '@wordpress/data';`.

Reply:
Good idea :)